pkg/ext-proc/metrics/testdata/request_total_metric

```diff
@@ -1,4 +1,4 @@
-# HELP inference_model_request_total [ALPHA] Counter of inference model requests broken out for each model and target model.
```

Comment:
Why we are removing this? This is to match the name.

Reply:
Added.